cql3/Cql.g

```diff
@@ -1007,13 +1012,14 @@ alterTableStatement returns [std::unique_ptr<alter_table_statement> expr]
             | '('     id1=cident { column_changes.emplace_back(alter_table_statement::column_change{id1}); }
                  (',' idn=cident { column_changes.emplace_back(alter_table_statement::column_change{idn}); } )* ')'
             )
+            ( K_USING K_TIMESTAMP ts=intValue { attrs->timestamp = std::move(ts); } )?
```

Comment:
can one use a parameter marker here? No? Is this documented?

Reply:
yes, intValue is:
```
intValue returns [uexpression value]
    : t=INTEGER     { $value = untyped_constant{untyped_constant::integer, $t.text}; }
    | e=marker      { $value = std::move(e); }
    ;
```
I don't know if this is documented in this patch (it should). Cassandra doesn't document the ability to use USING TIMESTAMP on a DROP at all, as far as I found (but maybe I missed something).

Reply:
Yeah, I'm missing documentation here, I'll add it.
Cassandra didn't documented this anyhow